examples/object_detection_3d/object_detection_3d/preprocess.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2021-2023 Kolena Inc.
```

Comment:
Would suggest moving this and the other scripts into a `scripts` folder to keep the integration clean. Our main goal with this example is to help users understand how to write an integration for a challenging and complex workflow — our objective is to make it as straightforward to understand and work with as possible.

Reply:
Will do. But side note, this file should not have been included, thanks for highlighting.